haiku/src/implementation.cc

```diff
@@ -1,10 +1,10 @@
 #include "iana-time-zone-haiku/src/interface.h"
 #include "iana-time-zone-haiku/src/lib.rs.h"
 
-#ifdef __HAIKU__
-
 #include <cstring>
 
+#ifdef __HAIKU__
```

Comment:
Why did you move `#include <cstring>` outside of the guarded block?

Reply:
because `size_t` is defined there which is needed for the `#else` block.

Reply:
gotcha. I undid these changes.